src/calculateRentalCost.js

```diff
@@ -4,7 +4,26 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
-}
+  let result = 0;
+  const DAY_COST = 40;
+  let discount = 0;
```

Comment:
Prefer using `const` for `discount` as it is not changed outside of the `for` loop scope. It should be redeclared within the loop body each iteration if needed.